### PR TITLE
WT-12864 Fixes to wtperf to support backup restarting without populate

### PR DESCRIPTION
**Pull request by sueloverso**
…ing.

### CODE REVIEW

File: test/utility/backup.c [hunk 1]
```diff
@@ -274,6 +274,48 @@ __int_comparator(const void *a, const void *b)
...
+ *     Find the last valid backup id number. Can be used after a restart to determine backups based
+ *     on directory naming.
+ */
+void
+testutil_last_backup_id(int *last_full, int *last_incr)
+{
+#ifdef _WIN32
+    WT_UNUSED(last_full);
+    WT_UNUSED(last_incr);
+#else
```
Comment (ddanderson) [inline test/utility/backup.c:288]: What's our story for WIN32?  If we don't support it, or don't support backup testing on it, we should probably assert here.

File: test/utility/backup.c [hunk 2]
```diff
@@ -274,6 +274,48 @@ __int_comparator(const void *a, const void *b)
     return (*(int *)a - *(int *)b);
 }
 
+/*
+ * testutil_last_backup_id --
+ *     Find the last valid backup id number. Can be used after a restart to determine backups based
+ *     on directory naming.
+ */
+void
+testutil_last_backup_id(int *last_full, int *last_incr)
```
Comment (ddanderson) [inline test/utility/backup.c:283]: `last_incr` is not used by the caller.  Is that something that will be supported soon?  If not, we shouldn't introduce dead code.

**Review by ddanderson — COMMENTED** [review]
I have a couple questions to resolve before approving.  The rest looks good.

File: test/utility/backup.c (hunk 2 above)
Comment (sueloverso) [reply test/utility/backup.c:283]: Yes, good call. If it is ever needed, then it can be added.

**Review by ddanderson — APPROVED** [review]
LGTM!

**Pull request merged**